src/test/java/org/assertj/core/error/OptionalShouldContain_create_Test.java

```diff
@@ -71,4 +72,14 @@ public void should_create_error_message_with_optionallong() {
                                        "  <10L>\n" +
                                        "but did not.");
   }
+
+  @Test
+  public void should_create_error_message_for_different_instances() {
+    String errorMessage = shouldContainSame(Optional.of(new Integer(10)), new Integer(10)).create();
+    assertThat(errorMessage).isEqualTo("\nExpecting:\n" +
```

Comment:
I wondered about that. As a Windows user I actually ran into failing tests here, but figured this was the AssertJ-style. :wink:

The whole class uses `%n` now.